modules/mono/editor/bindings_generator.cpp

```diff
@@ -3925,6 +3996,8 @@ void BindingsGenerator::_initialize() {
 
 	_initialize_blacklisted_methods();
 
+	compat_singletons.insert("EditorInterface");
+
```

Comment:
I think this should be added in my PR, while the framework for it should be added here.

Reply:
That makes sense, I was testing it with your PR cherry-picked and left it there.